### PR TITLE
Update overview,architecture and get-started page

### PR DESCRIPTION
**Pull request by apc999**

### CODE REVIEW

File: docs/en/Getting-Started.md [hunk 1]
```diff
@@ -22,25 +22,16 @@ The guide will cover the following tasks:
...
-environment:
-* [Accelerating on-demand data analytics with Alluxio](https://alluxio.com/resources/accelerating-on-demand-data-analytics-with-alluxio)
-* [Accelerating data analytics on Ceph object storage with Alluxio](https://www.alluxio.com/blog/accelerating-data-analytics-on-ceph-object-storage-with-alluxio).
+**Note**  This guide is designed to start an Alluxio system with minimal setup on a single machine.
+To try Alluxio a distributed environment for big data workloads, request [a sandbox cluster](https://www.alluxio.org/sandbox/request) with Alluxio and Spark installed on AWS for free.
 
 ## Prerequisites
 
-* Mac OS X or Linux
+* MacOS X or Linux
```
Comment (ZacBlanco) [inline docs/en/Getting-Started.md]: Technically, nowadays it's just "MacOS" they've dropped the X. The last "X" release was El Capitan in 2015

https://en.wikipedia.org/wiki/MacOS#OS_X_10.11_El_Capitan

File: docs/en/Getting-Started.md [hunk 2]
```diff
@@ -321,14 +312,16 @@ $ ./bin/alluxio-stop.sh local
...
-Congratulations on completing the quick start guide for Alluxio! This guide covered how to 
+Congratulations on completing the quick start guide for Alluxio! This guide covered how to
 download and install Alluxio locally with examples of basic interactions via the Alluxio
 shell. This was a simple example on how to get started with Alluxio.
 
 There are several next steps available. Learn more about the various features of Alluxio in
 our documentation. The resources below detail deploying Alluxio in various ways,
 mounting existing storage systems, and configuring existing applications to interact with Alluxio.
 
+## Next Step
```
Comment (ZacBlanco) [inline docs/en/Getting-Started.md]: "Steps"

File: docs/en/Overview.md [hunk 3]
```diff
@@ -44,27 +41,15 @@ from over 200 institutions including
 [Tencent](https://www.tencent.com),
 [UC Berkeley](https://amplab.cs.berkeley.edu/),
 and [Yahoo](https://www.yahoo.com/).
-The project is the data layer of the Berkeley Data Analytics Stack ([BDAS](https://amplab.cs.berkeley.edu/bdas/))
-and is also part of the [Fedora distribution](https://fedoraproject.org/wiki/SIGs/bigdata/packaging).
+
 Today, Alluxio is deployed in production by
 [hundreds of organizations](https://www.alluxio.org/community/powered-by-alluxio)
-and runs on clusters exceeding 1,000 nodes.
+with largest deployment exceeding 1,000 nodes.
```
Comment (ZacBlanco) [inline docs/en/Overview.md]: I would probably say "with the largest deployments" or "with the largest deployment"

File: docs/en/Architecture-DataFlow.md [hunk 4]
```diff
@@ -67,14 +65,24 @@ with other components; it only responds to requests via RPC services.
 The leading master records all file system transactions to a distributed persistent storage
 to allow for recovery of master state information; the set of records is referred to as the journal.
 
-#### Standby Masters
+### Standby Masters
+
+Standby masters are launched on different servers to provide fault-tolerance when running Alluxio
```
Comment (ZacBlanco) [inline docs/en/Architecture-DataFlow.md]: To improve the grammar here I would probably say "the Alluxio master in HA mode", "an Alluxio master in HA mode", or "when running Alluxio in HA mode"

File: docs/en/Architecture-DataFlow.md [hunk 5]
```diff
@@ -67,14 +65,24 @@ with other components; it only responds to requests via RPC services.
...
+leading master after.
 
-Standby masters read journals written by the leading master to keep their own
-copies of the master state up-to-date. They also write journal checkpoints for
-faster recovery in the future. They do not process any requests from other
-Alluxio components.
+### Secondary Masters
 
-### Worker
+When running Alluxio master locally without HA mode, one can start a secondary master on the same
```
Comment (ZacBlanco) [inline docs/en/Architecture-DataFlow.md]: Grammar: "**the** Alluxio master locally"

File: docs/en/Architecture-DataFlow.md (hunk 4 above)
Comment (apc999) [reply docs/en/Architecture-DataFlow.md]: fixed

File: docs/en/Architecture-DataFlow.md (hunk 5 above)
Comment (apc999) [reply docs/en/Architecture-DataFlow.md]: fixed

File: docs/en/Getting-Started.md (hunk 2 above)
Comment (apc999) [reply docs/en/Getting-Started.md]: fixed

File: docs/en/Getting-Started.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ This quick start guide goes over how to run Alluxio on a local machine.
 The guide will cover the following tasks:
 
 * Download and configure Alluxio
-* Validating Alluxio environment
+* Validate Alluxio environment
```
Comment (calvinjia) [inline docs/en/Getting-Started.md]: `Validate the`

File: docs/en/Getting-Started.md [hunk 7]
```diff
@@ -22,28 +22,19 @@ The guide will cover the following tasks:
...
 
-**Note**  This guide is designed to start an Alluxio system with minimal setup. Alluxio
-performs best in a distributed environment for big data workloads, but this scenario is difficult
-to simulate on a single machine. The performance benefits of Alluxio are illustrated in
-the following whitepapers, which include further instructions for running Alluxio in a scaled-up
-environment:
-* [Accelerating on-demand data analytics with Alluxio](https://alluxio.com/resources/accelerating-on-demand-data-analytics-with-alluxio)
-* [Accelerating data analytics on Ceph object storage with Alluxio](https://www.alluxio.com/blog/accelerating-data-analytics-on-ceph-object-storage-with-alluxio).
+**Note**  This guide is designed to start an Alluxio system with minimal setup on a single machine.
+To try Alluxio in a distributed environment for more realistic workloads, request [a sandbox cluster](https://www.alluxio.org/sandbox/request) with Alluxio and Spark installed on AWS for free.
```
Comment (calvinjia) [inline docs/en/Getting-Started.md]: `environment for` -> `environment with`

File: docs/en/Overview.md [hunk 8]
```diff
@@ -24,13 +24,10 @@ HDFS, IBM Cleversafe, EMC ECS, Ceph, NFS, and Alibaba OSS.
...
 APIs and a global namespace to its upper layer data driven applications.
 
-The Alluxio project originated from [AMPLab](https://amplab.cs.berkeley.edu/software/),
-UC Berkeley (see [papers](https://www.alluxio.org/resources/white-papers)).
-It is open source under [Apache License 2.0](https://github.com/alluxio/alluxio/blob/master/LICENSE)
-and is deployed at a
-[wide range of leading companies in the world](https://www.alluxio.org/community/powered-by-alluxio).
-It is one of the fastest growing open source projects.
-In a span of five years, Alluxio has attracted more than [800 contributors](https://github.com/alluxio/alluxio/graphs/contributors)
+The Alluxio project originated from [AMPLab at UC Berkeley](https://amplab.cs.berkeley.edu/software/) (see [papers](https://www.alluxio.org/resources/white-papers)) as
```
Comment (calvinjia) [inline docs/en/Overview.md]: `the UC Berkeley AMPLab`

File: docs/en/Overview.md [hunk 9]
```diff
@@ -24,13 +24,10 @@ HDFS, IBM Cleversafe, EMC ECS, Ceph, NFS, and Alibaba OSS.
...
+the data layer of the Berkeley Data Analytics Stack ([BDAS](https://amplab.cs.berkeley.edu/bdas/)).
```
Comment (calvinjia) [inline docs/en/Overview.md]: `data access`?

File: docs/en/Overview.md [hunk 10]
```diff
@@ -120,13 +105,25 @@ To quickly get Alluxio up and running, take a look at our
 [Getting Started]({{ '/en/Getting-Started.html' | relativize_url }}) page,
 which explains how to deploy Alluxio and run examples in a local environment.
 
-## Downloads and More
+## Downloads and More Useful Resources
```
Comment (calvinjia) [inline docs/en/Overview.md]: `Downloads and Useful Resources`

File: docs/en/Overview.md [hunk 11]
```diff
@@ -120,13 +105,25 @@ To quickly get Alluxio up and running, take a look at our
...
 Each release comes with prebuilt binaries compatible with various Hadoop versions.
 [Building From Master Branch Documentation]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }})
 explains how to build the project from source
 code. Questions can be directed to our
-[User Mailing List](https://groups.google.com/forum/?fromgroups#!forum/alluxio-users). Users who
-can not access the Google Group may use its [mirror](http://alluxio-users.85194.x6.nabble.com/)
-Note that the mirror does not have information before May 2016.
+[User Mailing List](https://groups.google.com/forum/?fromgroups#!forum/alluxio-users)
+or our [Community Slack Channel](https://alluxio.org/slack).
+Users who
```
Comment (calvinjia) [inline docs/en/Overview.md]: strange linebreak?

File: docs/en/Getting-Started.md (hunk 6 above)
Comment (apc999) [reply docs/en/Getting-Started.md]: done

File: docs/en/Getting-Started.md (hunk 7 above)
Comment (apc999) [reply docs/en/Getting-Started.md]: done

File: docs/en/Overview.md (hunk 8 above)
Comment (apc999) [reply docs/en/Overview.md]: done

**Pull request merged**